spec/system/prior_authority/primary_quote_spec.rb

```diff
@@ -42,4 +42,5 @@
     click_on 'Save and come back later'
     expect(page).to have_content 'Your applications'
   end
+  # TODO: Add system tests for uploading files
```

Comment:
Joel had a great suggestion for TODOs - it would be good to get into the discipline of only adding a TODO if we can also add a ticket number for the ticket that will address the TODO. That way it's harder for them to linger indefinitely.